sass/styles/style-2/style-2.scss

```
@@ -92,6 +92,10 @@ body:not(.header-solid-background) .site-header {
 	}
 }
 
+.hs-bg .highlight-menu a {
```

```suggestion
.h-sb .highlight-menu a {
```

I think you have the wrong class name here? In #584 they are `h-sb` and `h-db`. I think `h-sb` is the correct one to use? Solves the issue in my testing.

---

Oh good catch! 🤦‍♀ This is what I get for changing the class name at the last second!

I've updated the class in 168dfca.